content/docs/gherkin/step-organization.md

```diff
@@ -10,8 +10,8 @@ polyglot:
 ---
 
 You can have all of your step definitions in one file, or in multiple files. When you start with your project, all your step definitions will probably be in one file.
-As your project grows, it will be easiest to split them into meaningful groups in different files.
-This will make your project easier to maintain.
+As your project grows, you can then split them into meaningful groups in different files.
```

Comment:
As your project grows, you should split your step definitions into meaningful groups in different files